### PR TITLE
[Makefile] coq configure prefix and add nostdlib make

### PR DESCRIPTION
**Pull request by Alizter**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,10 +41,10 @@ watch: coq_boot
 build-all: coq_boot
 	dune build $(DUNEOPT) @all
 
+
+COQ_CONFIGURE_PREFIX=$(PWD)/_build/install/default/
 coq/config/coq_config.ml:
```
Comment (ejgallego) [inline Makefile:46]: Are you sure this is going to work? If we don't call configure anymore, the config file will be generate on the build phase, and I'm not sure that's gonna pick the right variable.

Umm, there is a typo here by the way (should be `vendor/coq`)

Comment (Alizter) [reply Makefile:46]: @ejgallego I believe it is this variable that is picked. That is indeed a typo.

Comment (ejgallego) [reply Makefile:46]: I'm afraid you broke the build:
```
coqlib is at: /home/egallego/research/coq-lsp/_build/default/vendor/coq/../install/default/lib/coq
```
which is incorrect; the variable can't be possibly picked.

Please go back to standard configure, make lazy variable semantics is a hell.

Comment (ejgallego) [reply Makefile:46]: The error message Coq gives for bad coqlib is really terrible, it just does `exit 1` and prints to the console :S

Comment (Alizter) [reply Makefile:46]: I really wish we didn't have to do any configure for Coq :(

Comment (ejgallego) [reply Makefile:46]: Yes that's a bit PITA.

Comment (ejgallego) [reply Makefile:46]: Fixed the typo directly in master, I had noticed the other day but forgot to fix it and instead put the fix in some branch not yet merged.

**Pull request merged**